src/tests/lib/BediEmbed.test.ts

```diff
@@ -1,19 +1,13 @@
 import {BediEmbed} from '../../lib/BediEmbed';
 
 describe('BediEmbed', () => {
-  test('BediEmbed has Correct Footer', () => {
-    const {footer} = new BediEmbed();
+  test('BediEmbed has Correct Properties', () => {
+    const {footer, color, timestamp} = new BediEmbed();
     const now = Date.now();
 
+    expect(timestamp).toBe(now);
+    expect(color).toBe(3553599);
     expect(typeof footer).toBe('object');
     expect(footer!.text).toBe('For any concerns, contact a BediBot dev: Aadi, Carson, Joe, Sahil, & Zayd');
   });
-
-  test('BediEmbed has Correct Timestamp', () => {
-    const {timestamp} = new BediEmbed();
-
-    const now = Date.now();
-
-    expect(timestamp).toBe(now);
-  });
 });
```

Comment:
This is one of the few times I think something doesn't need to be tested, I would honestly just assert/expect that the embed != None